src/billing/components/marketplace/MarketplaceLink.tsx

```diff
@@ -1,18 +1,20 @@
 import React, {FC} from 'react'
+import {useSelector} from 'react-redux'
 import {
   Button,
   ComponentColor,
   ComponentSize,
   ButtonShape,
 } from '@influxdata/clockface'
 
-import {Marketplace} from 'src/types/marketplace'
+// Types
+import {Me} from 'src/client/unityRoutes'
 
 // Utils
-import {useBilling} from 'src/billing/components/BillingPage'
+import {getQuartzMe} from 'src/me/selectors'
 
 const buttonInfo = (
-  marketplace: Marketplace['shortName']
+  marketplace: Me['billingProvider']
```

Comment:
looks like button info has all these links hard coded. We have an endpoint for getting this information. `/api/v2/marketplace`; can that be used here instead of these hard codings in case they change? The idea is that we wouldnt have to make the same change in both quartz and the ui.

Reply:
Definitely. Pretty sure this is based on the Quartz UI's implementation. If that endpoint is the `marketplace` one, then we can change this up to make a request to that endpoint for that data. Did we want to make the `marketplace` endpoint take a "shortName" so that we can make a request based on a region?

Reply:
Yea this is how quartz implements it. But we dont want to now have 2 sources of truth for this information when it gets moved over to the ui bc marketplace link is still going to exist in quartz. We use it in onboarding pages. 

The /marketplace endpoint should need to take anything. It will use lookup the marketplace for the account for the user in the session.

Reply:
Aw, perfect so we can just use the marketplace API here. I'll update it